packages/charts-react-line/src/LineChart.story.js

```diff
@@ -32,5 +32,15 @@ storiesOf('LineChart', module)
       domainY={extent(data, (d) => d.close)}
       x={(d) => d.date}
       y={(d) => d.close}
+      grid={[
+        {
+          text: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
```

Comment:
🤔  I don't understand how do you convert those values to Date objects to build correct line from the stock.csv values:
```
date,close
24-Apr-07,93.24
25-Apr-07,95.35
26-Apr-07,98.84
27-Apr-07,99.92
30-Apr-07,99.80
```